web/src/components/MemoEditor.tsx

```diff
@@ -97,6 +100,27 @@ const MemoEditor = () => {
     };
   }, []);
 
+  useEffect(() => {
+    if (state.isShowEmojiPicker) {
```

Comment:
It might be more better to put this checker into `EmojiPicker` component, just like https://github.com/usememos/memos/blob/main/web/src/components/common/Selector.tsx#L40